modules/cli-options/src/main/scala/scala/cli/commands/VerbosityOptions.scala

```diff
@@ -12,7 +12,9 @@ final case class VerbosityOptions(
     verbose: Int @@ Counter = Tag.of(0),
   @HelpMessage("Interactive mode")
   @Name("i")
-    interactive: Option[Boolean] = None
+    interactive: Option[Boolean] = None,
+  @HelpMessage("Enable actionable diagnostics")
+    actions: Option[Boolean] = None
```

Comment:
any reason why we don't want those on by default?

Reply:
IMO we shouldn't enable it by default now, it still require a work, for example integration with Metals.